_posts/2019-07-15-mosquitto-mqtt.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+current: post
+cover: Image_5.png
+title: Using WoTT to Cryptographically Secure access between a Mosquitto Brokered MQTT Client and Server
+date: 2019-07-15 17:00:00
+category: tutorials
+author: Fiona McAllister
+tags: [Open Source]
+class: post-template
+---
+
+# Using WoTT to Secure Access to a Mosquitto MQTT Server
+
+## Introduction 
+
+Mosquitto is a lightweight message broker for MQTT. MQTT itself is a pub-sub messaging protocol that is particularly popular amongst IoT applications due to its optimisation for high-latency networks. Together, they are effective for IoT usage being low-power to run and adding a layer of security to IoT devices. 
+By default the connection between a Mosquitto run client and server is unencrypted. Mosquitto does have provisions for securing messages between client and server that are very easy to configure- though most of these options involve a password file.
```

Comment:
'involve a password file' -> 'involve using passwords' ?

Also - we should mention that using passwords without TLS ins hazardous as anyone with access to the network can potentially sniff the password.

Reply:
this is part of Mosquitto specifically- you place usernames and passwords into a `pwfile`

Reply:
Ah I see your point. Fair enough.